README.md

```diff
@@ -33,15 +46,25 @@ Are you having issues or feature requests? Feel free to [let us know](https://gi
 Learn more about our security policy [here](https://github.com/tomkerkhove/promitor/security/policy).
 
 ## Donate
-Promitor is fully OSS and built free-of-charge, however, if you appreciate my work you can do a small donation.
+
+Promitor is fully OSS and built free-of-charge, however, if you appreciate my work
+you can do a small donation.
 
 [![Donate](https://img.shields.io/badge/Donate%20via-GitHub-blue.svg?style=flat-square)](https://github.com/users/tomkerkhove/sponsorship)
 
 ## Get in touch
-Do you have a security issue to report or just want to privately contact me? Feel free to [write me an email](mailto:kerkhove.tom@gmail.com)
 
-## Thank you!
-We'd like to thank all the services, tooling & NuGet packages that support us - [Thank you](https://promitor.io/thank-you)!
+Do you have a security issue to report or just want to privately contact me? Feel
+free to [write me an email](mailto:kerkhove.tom@gmail.com)
+
+## Thank you
```

Comment:
Was the `!` on purpose removed?

Reply:
Yeah - it's disallowed by https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md026---trailing-punctuation-in-header. Unfortunately they don't really provide much of a reason for why certain rules are enforced.

I can just disable that rule and add the `!` back in if you want, or we could disable it for this one item. What's your thoughts?

Reply:
Nah that's ok, thanks for pointing to that repo!